internal/test/test.proto

```diff
@@ -0,0 +1,13 @@
+syntax="proto3";
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/wrappers.proto";
+
+package example;
+
+option go_package="github.com/ncrypthic/protoc-gen-gofullmethods/internal/test";
```

Comment:
Do we need to change the username `ncrypthic` here?

Reply:
will do :+1: